perf/roles/trex/tasks/main.yml

```diff
@@ -9,29 +9,47 @@
       - python3
       - rdma-core-devel
       - zlib-devel
+      - driverctl
     state: installed
   tags: install
 
+- name: Install kernel-devel
+  yum:
+    name: kernel-devel-{{ ansible_kernel }}
+    state: present
+  tags: install
+
+- name: Install Development Tools group
+  yum:
+    name: "@Development Tools"
+    state: present
+  tags: install
+
 - name: stop existing trex server
   command: pkill -x _t-rex-64
   ignore_errors: true
   tags: setup
 
-- name: clone trex
-  git:
-    repo: '{{trex_repo}}'
-    dest: '{{trex_dir}}'
-    version: '{{trex_tag}}'
-    force: true
+- name: Create TRex install directory
+  file:
+    path: "{{ trex_install_dir }}"
+    state: directory
+    mode: "0755"
+  tags: install
+
+- name: Download TRex release
+  get_url:
+    url: "{{ trex_release_url }}"
+    dest: "{{ trex_install_dir }}/{{ trex_release_filename }}"
+    validate_certs: no
```

Comment:
I don't like this but I know why you need it. The CA certificates are not available in RHEL 8. Could you possibly install the CA instead of ignoring ssl errors?

Reply:
I tried to create my own self-signed certificate using open-ssl. Please review it.

Reply:
I think there was a misunderstanding. See my other review.

Reply:
Updated ..